docs/locations.md

```diff
@@ -0,0 +1,168 @@
+# Backup Storage Locations and Volume Snapshot Locations
+
+Ark v0.10 introduces a new way of configuring where Ark backups and their associated persistent volume snapshots are stored.
+
+## Motivations
+
+In Ark versions prior to v0.10, the configuration for where to store backups & volume snapshots is stored in a `Config` custom resource. The `backupStorageProvider` section captures the place where all Ark backups should be stored. This is defined by a **provider** (e.g. `aws`, `azure`, `gcp`, `minio`, etc.), a **bucket**, and possibly some additional provider-specific settings (e.g. `region`). Similarly, the `persistentVolumeProvider` section captures the place where all persistent volume snapshots taken as part of Ark backups should be stored, and is defined by a **provider** and additional provider-specific settings (e.g. `region`).
```

Comment:
```suggestion
In Ark versions prior to v0.10, the configuration for where to store backups & volume snapshots is specified in a `Config` custom resource. The `backupStorageProvider` section captures the place where all Ark backups should be stored. This is defined by a **provider** (e.g. `aws`, `azure`, `gcp`, `minio`, etc.), a **bucket**, and possibly some additional provider-specific settings (e.g. `region`). Similarly, the `persistentVolumeProvider` section captures the place where all persistent volume snapshots taken as part of Ark backups should be stored, and is defined by a **provider** and additional provider-specific settings (e.g. `region`).
```

Reply:
Super minor change just to avoid repeating the concept of "storage" and perhaps avoid confusion.

Reply:
thanks, updated!